Sources/allonet2/PlaceServer.swift

```diff
@@ -318,16 +326,56 @@ public class PlaceServer : AlloSessionDelegate
     {
         switch inter.body
         {
-        case .announce(let version, let avatarDescription):
+
+        case .registerAsAuthenticationProvider:
+            // Reasons this is bad:
+            // - First wins
+            // - Only one provider per place server
+            // - No verification that the client is actually allowed to authenticate others
+            // - A client could authenticate itself
+            if authenticationProvider == nil {
+                authenticationProvider = client
+                client.session.send(interaction: inter.makeResponse(with: .success))
```

Comment:
How about a TODO (or even do the work if you can be bothered): 

`// TODO: authenticate all currently connected clients to make sure they're allowed by our new provider`